### ISSUE TITLE
Sensor logs exist in both warmup test split and training split

### ISSUE THREAD

**Issue by dentou** [autonomousvision/navsim#40]
Hello,

I noticed that the following logs in [warmup_test_e2e.yaml](https://github.com/autonomousvision/navsim/blob/main/navsim/planning/script/config/common/train_test_split/scene_filter/warmup_test_e2e.yaml) are presented as `train_logs` in [default_train_val_test_log_split.yaml](https://github.com/autonomousvision/navsim/blob/main/navsim/planning/script/config/training/default_train_val_test_log_split.yaml). I'm currently testing on the `mini` dataset. Does this meaning that these logs are used in both training and testing? 

Thanks in advance,

```
2021.05.12.22.00.38_veh-35_01008_01518
2021.05.12.22.28.35_veh-35_00620_01164
2021.05.12.23.36.44_veh-35_00152_00504
2021.05.12.23.36.44_veh-35_01133_01535
2021.05.12.23.36.44_veh-35_02035_02387
2021.06.09.11.54.15_veh-12_04366_04810
2021.06.09.12.39.51_veh-26_01943_02303
2021.06.09.12.39.51_veh-26_05620_06003
2021.06.09.14.03.17_veh-12_02584_02970
2021.06.09.14.58.55_veh-35_01095_01484
2021.06.09.14.58.55_veh-35_01894_02311
2021.06.09.17.23.18_veh-38_00773_01140
2021.06.09.17.23.18_veh-38_02526_03027
2021.06.09.17.37.09_veh-12_00404_00864
2021.06.14.16.32.09_veh-35_05038_05402
2021.06.14.16.48.02_veh-12_04057_04438
2021.06.14.16.48.02_veh-12_04978_05337
2021.06.14.17.26.26_veh-38_04544_04920
2021.06.14.18.33.41_veh-35_03901_04264
2021.06.14.18.42.45_veh-12_03445_03902
2021.06.14.19.22.11_veh-38_01480_01860
2021.06.23.15.56.12_veh-16_00839_01285
2021.06.23.16.54.19_veh-35_00808_01256
2021.06.23.17.31.36_veh-16_00016_00377
2021.07.09.17.06.37_veh-35_00258_00748
2021.07.09.20.59.12_veh-38_01208_01692
2021.07.16.00.51.05_veh-17_01352_01901
2021.07.16.18.06.21_veh-38_03231_03712
2021.07.16.18.06.21_veh-38_04471_04922
2021.07.16.18.06.21_veh-38_04933_05307
2021.07.16.18.19.22_veh-35_00440_00858
2021.07.16.20.45.29_veh-35_00600_01084
2021.07.16.20.45.29_veh-35_01095_01486
2021.08.17.16.57.11_veh-08_01200_01636
2021.08.17.17.17.01_veh-45_02314_02798
2021.08.17.18.54.02_veh-45_00665_01065
2021.10.01.19.16.42_veh-28_02011_02410
2021.10.01.19.16.42_veh-28_03307_03808
2021.10.06.17.43.07_veh-28_00508_00877
2021.10.11.02.57.41_veh-50_00352_00535
2021.10.11.02.57.41_veh-50_01522_02088
2021.10.11.07.12.18_veh-50_00211_00304
2021.10.11.08.31.07_veh-50_01750_01948
```

**Comment by mh0797** [comment]
Hi @dentou,

the warmup-test split is indeed based on the mini split - not only the test logs within mini. Thus, there can be overlaps between the mini train logs and the warmup test ones.
Warmup-test‘s mere purpose was to give participants an opportunity to get familiar with the submission procedure. As such, it was not meant as a competitive benchmark to evaluate performance. 

After the competition we launched the public leaderboard based on navtest. I encourage you to use the navtest split as a held out test split instead of warmup_test. Since the open leaderboard has less strict limits regarding the number of submissions, you can also directly submit there and there is no need start by running experiments on mini.


(I am closing this for now. Feel free to re-open)

**Comment by dentou** [comment]
Hi @mh0797,
Thanks for the quick response. I think we can update [this answer](https://github.com/autonomousvision/navsim/issues/11#issuecomment-2073451195). In particular, my misunderstanding came from this section

> **test-logs in mini vs. warmup_test_e2e**
The warmup_test_e2e scene-filter filters the test-logs from the mini dataset for challenging scenarios by employing the same strategy that is used to generate navtrain from trainval. Thus it subsamples these logs.

Since the test set is ~50% more than the mini set, I think a test split on mini could be helpful as a initial sanity check. Anyway, thanks for the suggestion about the navtest split.

**Comment by mh0797** [comment]
Hi,
thanks for pointing this out. I updated the answer accordingly. 

You can always reduce the number of scenarios by setting `train_test_split.scene_filter.max_scenes=<some_value>`
